src/utils/selectedCellUtils.ts

```diff
@@ -96,7 +99,6 @@ export function getNextSelectedCellPosition<R, SR>({
           idx: 0
         };
       }
-      setColSpan(true);
```

Comment:
It is not required as the next position is going to be the first cell of a colSpan range